inc/class-markdownparser.php

```diff
@@ -54,15 +54,22 @@ protected function inlineLink( $data ) {
 
 		// Resolve relative to the current file.
 		$base = $this->current_page->get_meta( 'path' );
-		$root= $this->current_page->get_meta( 'root' );
+		$root = $this->current_page->get_meta( 'root' );
 		$resolved = realpath( path_join( dirname( $base ), $href ) );
 		if ( empty( $resolved ) ) {
 			return $result;
 		}
 
 		// Override href.
-		$slug = get_slug_from_path( $root, $resolved );
-		$url = get_url_for_page( UI\get_current_group_id(), $slug );
+
+		// Support URLs to assets directories in a module's docs directory.
+		if ( strpos( $href, './assets/' ) === 0 ) {
```

Comment:
Does this mean the image URL in the markdown always needs to start with `./assets/`? There might be cases where the file is a level or 2 deeper and would start `../assets/`, or even use a relative path like `assets/image.png`